### DISCUSSION TITLE
Māori

### DISCUSSION THREAD

**Discussion by spongefile** [kantord/LibreLingo#3392] [Ideas]
Duolingo was supposed to come out with a Māori module years ago but it never appeared. There's an increasing demand for learning resources for this language, of which there are very few. If contribution can be made as layperson-accessible as possible, this could well be a fruitful language to start with.

**Comment by Skivling** [comment]
Te Reo Māori has recently been started: https://codeberg.org/Skivling/LibreLingo-MI-from-EN

We have a Matrix chat room: https://matrix.to/#/#tereo-librelingo:mtrx.nz

It will be added to LibreLingo after module 1 has been completed.
